### ISSUE TITLE
Get TN from targeted token

### ISSUE THREAD

**Issue by fougerec** [jalensailin/mvrpg#33]
When I target an enemy it doesn't seem to take into account what their defense is and just says "11".
![image](https://github.com/jalensailin/mvrpg/assets/84296981/a42895b4-5d6f-4ff6-bf8e-8cb79283b857)
![image](https://github.com/jalensailin/mvrpg/assets/84296981/d441e1fc-30d1-4a84-b04a-442eb24d20a0)

I've tried this with existing characters and newly made ones.

**Comment by jalensailin** [comment]
Correct, getting the TN from a targeted token is not yet supported. The TN defaults to the "Challenging" target for the rolling actor's rank, but otherwise must be updated manually, for now
